project/ISSUE-TRIAGE.md

```diff
@@ -0,0 +1,109 @@
+## Triaging of Issues
+
+Docker has a lot of issues and it can be sometimes daunting to look through
+everything to find something to work on.  Triaging issues is important to
+help both the author convey the real problem or request that they have and
+to help developers easily find something to work on.
+
+Remember there are more ways than just code that you can contribute to an
+open source project and the community would really thank you if you spent
+spend time triaging issues.
+
+
+### Important first steps
+
+Before jumping into an issue very far make sure that the author of the issue
+has provide enough information so that you are able to make an educated 
+reccomendation on how this is to be triaged.  Standard information that *must*
+be included in most issues are things such as:
+
+* docker version
+* docker info
+* host distro ( ubuntu, RHEL, fedora )
+
+A bonus is to ask for a way to reproduce the issue.
+
+Additional information can be requested if needed.  If an issue is totally unable
+to be triaged based on the author's provided information, kindly state that
+they will have to provide the above information and clarification on the problem.
+Close the issue letting them know they can re-open once the information is 
+available so that it is able to be properly triaged.  Closing the issue is ok
+because some people do not have time to respond quickly and there is no use
+leaving an issue open that cannot be acted upon.  Feel free to point them to 
+this document so that they understand the issue is not being closed due to
+maintainers and contributors not wanting to help but because they don't have
+the information to provide help in a meaningful way.
+
+
+### Classification
```

Comment:
You may want to move the Template section up to here. By getting into the Classifications w/o letting us know why or where they would appear it feels out of place.  I'd talk about how new issues should follow a certain template - show the template and then explain each field/section of it.  Should set the context a little better.